examples/move/scratch_off/sources/math.move

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module scratch_off::math {
+
+    /// Hard coding this as 1 billion
+    const MODULO_FACTOR: u256 = 1_000_000_000;
+
+    public fun bytes_to_u256(hashed_beacon: &vector<u8>): u256 {
+        let output = 0;
+        let bytes_length = 32;
+        let idx = 0;
+        while (idx < bytes_length) {
+            let current_byte = *std::vector::borrow(hashed_beacon, idx);
+            output = (output << 8) | (current_byte as u256) ;
+            idx = idx + 1;
+        };
+        output
+    }
+
+    public fun get_result(random_number: u256): u64 {
+        ((random_number % MODULO_FACTOR) as u64)
+    }
+
+    public fun get_random_u64(hashed_beacon: &vector<u8>): u64 {
```

Comment:
The naming of this method is a bit confusing. Something like `get_random_modulo_factor` would be better, i think. It would also make sense to add a doc saying that the vector should be at least 32 bytes.

Reply:
Modified the module factor and removed it